### ISSUE TITLE
Training code is different with paper

### ISSUE THREAD

**Issue by agusgun** [YesianRohn/TextSSR#8]
Hi, I have seen your code and based on the batch size in the code (8), the batch size in the paper is 512 (8GPUs mean 64 per GPU, training batch size should be 64). Can I know if this difference is expected? Also, regarding the training steps, the paper say it's 150000, but your code use 100 training epochs.

In addition, the training code for VAE seems slow to run.

**Comment by YesianRohn** [comment]
For your first question, this is because gradient_accumulation_steps=8 is used to achieve gradient accumulation.  This means that although the per-GPU batch size is set to 8, with 8 accumulation steps, the effective batch size per GPU becomes 64, matching the settings described in the paper.

For the second question, the setting of 100 training epochs is just a maximum preset value in the code.  In practice, the model is trained for 150,000 steps as mentioned in the paper, and not necessarily for the full 100 epochs.  Therefore, running for 150,000 steps should actually take less time than training for 100 epochs.
